### PR TITLE
Support logical_count command

### PR DESCRIPTION
**Pull request by cosmo0920**
#2

### CODE REVIEW

File: test/command/test-logical-count.rb [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                     :min_border     => min_border,
+                     :max            => max,
+                     :max_border     => max_border,
+                     :filter         => filter,
+                   },
+                   command.arguments)
+    end
+  end
+
+  class TableTest < self
```
Comment (kou) [inline test/command/test-logical-count.rb]: It should be `LogicalTableTest`.

File: test/command/test-logical-count.rb [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    def test_reader
+      command = logical_count_command(:logical_table => "Logs")
+      assert_equal("Logs", command.logical_table)
+    end
+  end
+
+  class ColumnTest < self
```
Comment (kou) [inline test/command/test-logical-count.rb]: It should be `ShardKeyTest`.

**Comment by cosmo0920** [comment]
I've fixed it!

**Comment by kou** [comment]
Thanks!
I've merged.

**Comment by cosmo0920** [comment]
Thank you for merging!

**Pull request merged**